apps/web-connect/src/utils/convertEthAmountToUsdcAmount.ts

```diff
@@ -0,0 +1,30 @@
+import { getEthUsdcExchangeRate } from "@sentry/core";
+
+/**
+ * Converts an amount of ETH (in wei) to its equivalent value in USDC or another specified precision.
+ * 
+ * @async
+ * @param {bigint} ethAmountInWei - The amount of ETH in wei (18 decimal places).
+ * @param {number} [outputDecimals=6] - The number of decimal places for the output (default is 6, for USDC precision).
+ * @returns {Promise<bigint>} The equivalent amount in the specified decimal precision.
+ * 
+ */
+export const convertEthAmountToUsdcAmount = async (
+    ethAmountInWei: bigint,
+    outputDecimals: number = 6
```

Comment:
Why do we need `outputDecimals`, should this not just return either a wei amount or an eth amount ?

Reply:
In order to keep our UI display accurate, in some places, we need to scale up the USDC amount to 18 decimals, or else the U/I display breaks for USDC balances etc...

Reply:
Oh my....